GNUmakefile

```diff
@@ -30,9 +28,8 @@ lint:
 
 tools:
 	@echo "==> installing required tooling..."
-	go install github.com/client9/misspell/cmd/misspell
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint
-
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell go env GOPATH)/bin v1.18.0
+	curl -L https://git.io/misspell | BINDIR=$(shell go env GOPATH)/bin bash
```

Comment:
Fetching the compiled binaries means we don't have to build them from source ourselves, which means we don't inherit their dependencies.

Reply:
Hi @ryboe! Would you be able to provide me steps to reproduce `make tools` failing?  I've cloned a clean repo and run it and it still succeeds for me. Thanks!

Reply:
My mistake! `make tools` is working fine. I'm not sure why it was failing earlier for me. I'll close this PR. Thanks for catching this 🙂